posts/git-cherry-pick/index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Cherry-Pick on GitHub"
+author: "Robyn Marowitz"
+date: "2023-09-26"
+description: "Committing a mistake in GitHub can feel so scary, cherry picking is a tool to help you fix your history when something like that happens. This post lays out simple and clear instructions on the cherry picking method."
+categories: [github, open-science]
+image: "cherry_pick.jpeg"
+---
+
+## What is git cherry-pick?
+
+It is the act of picking a commit from one branch and applying it to another branch.  
+
+## When should I use cherry-pick?
+
+Cherry-Pick is best used when a change has been committed onto a different branch than desired. You can *pick* commits one by one and move them to the desired branch. 
```

Comment:
The official docs don't capitalize each word. The use of hyphenation is inconsistent. Maybe there's logic to it, but I'm not seeing it :laughing:  e.g.:

>When recording the commit, append a line that says "(cherry picked from commit …​)" to the original commit message in order to indicate which commit this change was cherry-picked from. This is done only for cherry picks without conflicts. Do not use this option if you are cherry-picking from your private branch because the information is useless to the recipient. If on the other hand you are cherry-picking between two publicly visible branches (e.g. backporting a fix to a maintenance branch for an older release from a development branch), adding this information can be useful.

Reply:
>can *pick* commits one by one

You can also pick ranges of commits in one go, e.g. `git cherry-pick efefefe..fefefef`